src/test/javascript/spec/integration/code-editor/code-editor-container.integration.spec.ts

```diff
@@ -354,14 +330,15 @@ describe('CodeEditorContainerIntegration', () => {
 
         containerFixture.detectChanges();
 
-        expect(container.selectedFile).to.equal(selectedFile);
-        expect(container.aceEditor.selectedFile).to.equal(selectedFile);
-        expect(container.aceEditor.isLoading).to.be.false;
-        expect(Object.keys(container.aceEditor.fileSession)).to.contain(selectedFile);
-        expect(getFileStub).to.have.been.calledOnceWithExactly(selectedFile);
+        expect(container.selectedFile).toEqual(selectedFile);
+        expect(container.aceEditor.selectedFile).toEqual(selectedFile);
+        expect(container.aceEditor.isLoading).toBe(false);
+        expect(Object.keys(container.aceEditor.fileSession)).toContain(selectedFile);
+        expect(getFileStub).toHaveBeenCalledTimes(1);
+        expect(getFileStub).toHaveBeenCalledWith(selectedFile);
 
         containerFixture.detectChanges();
-        expect(container.aceEditor.editor.getEditor().getSession().getValue()).to.equal(fileContent);
+        expect(container.aceEditor.editor.getEditor().getSession().getValue()).toEqual(fileContent);
```

Comment:
As you are checking against a string, I would recommend `toBe()`
```suggestion
        expect(container.aceEditor.editor.getEditor().getSession().getValue()).toBe(fileContent);
```